src/HW_models/NRF_RADIO.c

```diff
@@ -368,6 +368,19 @@ void nrf_radio_regw_sideeffects_TASKS_DISABLE(){
   }
 }
 
+void nrf_radio_regw_sideeffects_TASKS_RSSISTART() {
```

Comment:
If you don't need it, and this way it doesn't work, would it be better to print the warning and do nothing otherwise? (not call `nrf_radio_tasks_rssistart()/stop();`)